packages/nimble-components/src/wafer-map/specs/README.md

```diff
@@ -0,0 +1,235 @@
+As a first step, we are working on the "bloat removal" which is basically the removal of unwanted but exiting features (legend, tooltip etc.) and dependencies.
+
+Original component: [wafermap](https://dev.azure.com/ni/DevCentral/_git/op-web-portal?path=/libraries/wafermap&version=GBfeature/microServicesSplit)\
+Once the "bloat removal" is finished, the next step is to adapt the current
+
+### Code submission strategy
+
+In order to make the process of submitting the re-written wafer map component easier for us and for the code reviewers as well, we propose the following bottom-up approach:
+
+-   As an initial step, we submit a skeleton of the `nimble-wafer-map` component into the main branch. This will serve as a starting point on which the complete functionality will be built on.
+-   Once the skeleton is in place, we start to work on and submit individual pieces of the functionality so it's not very overwhelming to review the code.
+
+### States
+
+We are not planning to allow the user to generate an “empty" wafer map, so we must ensure that data is provided when instantiating a new `nimble-wafer-map` component.
+The only action which we could consider as a "state transition" is the zoom functionality which was specified in the sections above.
+We expect that the end user be able to zoom in on the dies within the wafer map and be able to zoom out by using the mouse wheel.
+Also, the user must be allowed to pan the zoomed in wafer map into any direction of its XY axis by holding the left mouse button and moving the mouse cursor around.
+
+Another interaction, which must be part of the component is the ability to identify the location of the mouse pointer on the wafer map die area.
+
+Furthermore `disabled` and `focused` state will also be supported by the component.
+
+### Accessibility
+
+Important: this is out of scope for the current implementation.\
+Since the `nimble-wafer-map` component was designed to display tens of thousands of data points, it's challenging to provide Accessibility functionality which can read out loud the values displayed in a meaningful manner.
+Based on the WAI documentation on [Complex Images](https://www.w3.org/WAI/tutorials/images/complex/), in the future we could provide a paragraph containing information about the name of the wafer that is displayed, the LOT where the wafer comes from and a general overview (e.g. % of the faulty dies on the wafer) of the status. This might give a pretty good description about what the displayed nimble component shows to the user.
+Please note that in the initial implementation we do not plan to gather and summarize any of this information so this will be part of future updates to this component.
+Currently the only way to navigate and interact with the wafer map is only by using the mouse. Future updates might include functionality which allows the user to tab into the wafer map area and move around in the die area using the keyboard arrows.
```

Comment:
We can choose to make the accessibility good in a follow-up but we should do the minimum to make sure we don't break accessibility of existing pages today.

I think we should:
- Make sure we have tab focus states (as already described above)
- Make sure to choose a good role to represent the visual. I think [`role="img"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role) or [`role="presentation"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) are likely good and I think I lean towards `img` role

Reply:
Thanks @rajsite. We will go with your suggestion on this.